starters/pom.xml

```
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>starters</artifactId>
+    <name>starters</name>
+    <packaging>pom</packaging>
+    <description>miranum starters</description>
+
+    <parent>
+        <groupId>io.miragon.miranum</groupId>
+        <artifactId>miranum</artifactId>
```

```suggestion
        <artifactId>miranum-connect</artifactId>
```
⛏️ This is currently being changes in https://github.com/FlowSquad/miranum/pull/94. It would be great if we can change it in here right away. (This would require https://github.com/FlowSquad/miranum/pull/94 to be merged so we can do a rebase)

---

So let's merge #94 asap, will rebase on main branch then

---

I'm sure @DaAnda97 will do so after the weekend! :)